resources/migrations/000_migrations.yaml

```diff
@@ -14300,6 +14300,61 @@ databaseChangeLog:
             constraintName: fk_action_model_id
             onDelete: CASCADE
 
+  - changeSet:
```

Comment:
This is tested here https://github.com/metabase/metabase/pull/28849/files#diff-41093075df90623837726c733afc60b57899a66d6e9878b4250413b74f045f67R967